test/debugger/test-debugger-stop.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+var debug = require('_debugger');
+
+var fork = require('child_process').fork;
+
+var scriptToDebug = common.fixturesDir + '/debug-connect.js';
+
+var nodeProcess;
+
+try {
+  process._debugProcess(process.pid);
+} catch(ex) {
+  assert.fail('Debugging should not fail');
+}
+
+nodeProcess = fork(scriptToDebug);
+nodeProcess.send({ pid: process.pid });
+
+nodeProcess.on('message', common.mustCall(function(m) {
+  process._debugEnd();
+  process.exit();
```

Comment:
Try very hard to avoid calling `process.exit()` in a test, it has a history of hiding test failures.